src/solvers/uniswap/uniswapSolver.ts

```diff
@@ -188,14 +183,24 @@ export class UniswapSolverAdapter extends SolverAdapter {
             venue: "propeller",
         };
         const response = await axios.post(QUOTER_API_URL, query);
-        const quotes: Quote[] = [];
-        for (const quote of response.data.quotes) {
-            quotes.push({
-                sell_token: quote.quotes[0].sell_token,
-                buy_token: quote.quotes[0].buy_token,
-                sell_amount: quote.quotes[0].sell_amount,
-                buy_amount: quote.quotes[0].buy_amount,
-            });
+        const quotes: Quote[][] = [];
+        for (let i = 0; i < response.data.quotes.length; i += 2) {
+            const minQuote = response.data.quotes[i];
+            const maxQuote = response.data.quotes[i + 1];
+            quotes.push([
+                {
+                    sell_token: minQuote.quotes[0].sell_token,
+                    buy_token: minQuote.quotes[0].buy_token,
+                    sell_amount: minQuote.quotes[0].sell_amount,
+                    buy_amount: minQuote.quotes[0].buy_amount,
+                },
+                {
+                    sell_token: maxQuote.quotes[0].sell_token,
+                    buy_token: maxQuote.quotes[0].buy_token,
+                    sell_amount: maxQuote.quotes[0].sell_amount,
+                    buy_amount: maxQuote.quotes[0].buy_amount,
+                },
+            ]);
```

Comment:
Won't this do the same thing?
```suggestion
            quotes.push([minQuote.quotes[0], maxQuote.quotes[0]]);
```